test/governance/utils/Votes.behavior.js

```diff
@@ -255,28 +256,29 @@ function shouldBehaveLikeVotes () {
 
       it('returns zero if < first checkpoint block', async function () {
         await time.advanceBlock();
-        const t2 = await this.votes.mint(this.account1, this.NFT1);
+        const t2 = await this.votes.mint(this.account1, this.token1);
         await time.advanceBlock();
         await time.advanceBlock();
+        const block = await time.latestBlock();
 
         expect(await this.votes.getPastTotalSupply(t2.receipt.blockNumber - 1)).to.be.bignumber.equal('0');
-        expect(await this.votes.getPastTotalSupply(t2.receipt.blockNumber + 1)).to.be.bignumber.equal('1');
```

Comment:
Why was it necessary to change this? The previous code looks to me like it should've worked.

Reply:
You are right it works as it was before, I must have been testing something and then drop it, forgetting to recall my update.